coala_quickstart/generation/Settings.py

```diff
@@ -61,7 +61,7 @@ def generate_ignore_field(project_dir, languages, extset, ignore_globs):
     for glob in ignore_globs:
         gitignore_files = {file
                            for file in collect_files([glob], null_printer)}
-        if not all_files.isdisjoint(gitignore_files):
+        if all_files.isdisjoint(gitignore_files):
```

Comment:
why did you remove `not`? If set of `all_files` is entirely different from set of `gitignore_files`, we shouldn't add ignore field.